### PR TITLE
Translate /docs/topics/kubernetes_distros in Korean

### PR DESCRIPTION
**Pull request by bluefriday**
Signed-off-by: bluefriday <bluefriday86@gmail.com>

Translate /docs/topics/kubernetes_distros in Korean
- https://helm.sh/docs/topics/kubernetes_distros/

### CODE REVIEW

**Comment by bluefriday** [comment]
@jmnote @jmyung 리뷰 부탁드립니다.

File: content/ko/docs/topics/kubernetes_distros.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+Hyperkube 의 경우 몇 가지 수동 구성을 해야 할 수도 있다.
+
+## IKS
+
+헬름은 [IBM 클라우드 쿠버네티스 서비스](https://cloud.ibm.com/docs/containers?topic=containers-helm)에서 
+동작한다.
+
+## KIND (Kubernetes IN Docker)
+
+헬름은 [KIND](https://github.com/kubernetes-sigs/kind)에서 정기적으로 테스트 된다.
```
Comment (jmnote) [inline content/ko/docs/topics/kubernetes_distros.md:50]:
```suggestion
## KIND (Kubernetes IN Docker)

헬름은 [KIND](https://github.com/kubernetes-sigs/kind)에서 정기적으로 테스트된다.
```

File: content/ko/docs/topics/kubernetes_distros.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+## KubeOne
+
+헬름은 주의요소 없이 KubeOne 으로 구성된 클러스터에서 동작한다.
```
Comment (jmnote) [inline content/ko/docs/topics/kubernetes_distros.md:54]:
```suggestion
## KubeOne

헬름은 별다른 특이사항 없이 KubeOne 으로 구성된 클러스터에서 동작한다.
```

File: content/ko/docs/topics/kubernetes_distros.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+## Kubermatic
+
+헬름은 주의요소 없이 Kubermatic 으로 생성된 사용자 클러스터에서 동작한다.
+시드 클러스터는 다양한 방식으로 설정될 수 있으므로 헬름 지원은 
+그 설정에 따라 달라진다.
```
Comment (jmnote) [inline content/ko/docs/topics/kubernetes_distros.md:60]:
```suggestion
## Kubermatic

헬름은 별다른 특이사항 없이 Kubermatic 으로 생성된 사용자 클러스터에서 동작한다.
시드(seed) 클러스터는 다양한 방식으로 설정될 수 있으므로 헬름 지원은 
그 설정에 따라 달라진다.
```

File: content/ko/docs/topics/kubernetes_distros.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+## MicroK8s
+
+헬름은 `microk8s.enable helm` 명령을 사용하여 [MicroK8s](https://microk8s.io) 에서 
+사용가능하도록 활성화 할 수 있다.
```
Comment (jmnote) [inline content/ko/docs/topics/kubernetes_distros.md:65]:
```suggestion
## MicroK8s

[MicroK8s](https://microk8s.io) 에서는 명령어 `microk8s.enable helm`를 사용하여 헬름을 활성화할 수 있다.
```

File: content/ko/docs/topics/kubernetes_distros.md [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+
+## Minikube
+
+헬름은 테스트를 거쳐, [Minikube](https://github.com/kubernetes/minikube) 에서 
+동작하는 것으로 알려져있다. 
+추가적인 구성이 필요하지 않다.
```
Comment (jmnote) [inline content/ko/docs/topics/kubernetes_distros.md:71]:
```suggestion
## Minikube

헬름은 [Minikube](https://github.com/kubernetes/minikube) 에서 테스트를 거쳤고
동작하는 것으로 알려져 있다. 
추가적인 구성이 필요하지 않다.
```

File: content/ko/docs/topics/kubernetes_distros.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+
+## Openshift
+
+헬름은 OpenShift Online, OpenShift Dedicated, OpenShift Container Platform (버전 3.6 이상) 또는 
+OpenShift Origin (버전 3.6 이상)에서 동작한다.
+자세한 내용은 
+[이 블로그](https://blog.openshift.com/getting-started-helm-openshift/) 의 게시물을 참조한다.
```
Comment (jmnote) [inline content/ko/docs/topics/kubernetes_distros.md:78]:
```suggestion
## Openshift

헬름은 OpenShift Online, OpenShift Dedicated, OpenShift Container Platform (버전 3.6 이상) 또는 
OpenShift Origin (버전 3.6 이상)에서 동작한다.
자세한 내용은 
[이 블로그](https://blog.openshift.com/getting-started-helm-openshift/) 의 게시물을 참조하자.
```

File: content/ko/docs/topics/kubernetes_distros.md [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+
+## Platform9
+
+헬름은 [Platform9 Managed
+Kubernetes](https://platform9.com/managed-kubernetes/?utm_source=helm_distro_notes)와 함께 사전 설치된다.
+Platform9 은 앱 카탈로그 UI 및 기본 쿠버네티스 CLI를 통해 
+모든 공식 헬름 차트에 대한 접근을 제공한다. 추가적인 레포지토리들도 수동으로 추가할 수 있다.
+더 상세한 내용은 [Platform9 App Catalog
+article](https://platform9.com/support/deploying-kubernetes-apps-platform9-managed-kubernetes/?utm_source=helm_distro_notes)에서 확인할 수 있다.
```
Comment (jmnote) [inline content/ko/docs/topics/kubernetes_distros.md:87]:
```suggestion
## Platform9

헬름은 [Platform9 Managed
Kubernetes](https://platform9.com/managed-kubernetes/?utm_source=helm_distro_notes)와 함께 사전 설치된다.
Platform9 은 앱 카탈로그 UI 및 기본 쿠버네티스 CLI를 통해 
모든 공식 헬름 차트에 대한 접근을 제공한다. 추가적인 레포지토리들도 수동으로 추가할 수 있다.
상세한 내용은 [Platform9 App Catalog
article](https://platform9.com/support/deploying-kubernetes-apps-platform9-managed-kubernetes/?utm_source=helm_distro_notes)에서 확인할 수 있다.
```

File: content/ko/docs/topics/kubernetes_distros.md [hunk 8]
```diff
@@ -0,0 +1,98 @@
...
+## Ubuntu with `kubeadm`
+
+`kubeadm` 으로 부트스트랩 된 쿠버네티스는 Linux 배포버전에서 
+동작하는 것으로 알려져 있다.
+
+- Ubuntu 16.04
+- Fedora release 25
+
+헬름의 일부 버전(v2.0.0-beta2)은 `export KUBECONFIG=/etc/kubernetes/admin.conf` 를 수행하거나
+`~/.kube/config` 를 생성해야 한다.
```
Comment (jmnote) [inline content/ko/docs/topics/kubernetes_distros.md:98]:
```suggestion
## 우분투와 `kubeadm`

`kubeadm` 으로 구성된(bootstrapped) 쿠버네티스는 다음 리눅스 배포판에서 
동작하는 것으로 알려져 있다.

- 우분투 16.04
- 페도라 릴리스 25

헬름 일부 버전(v2.0.0-beta2)은 `export KUBECONFIG=/etc/kubernetes/admin.conf` 를 수행하거나
`~/.kube/config` 를 생성해야 한다.
```

**Review by jmnote — COMMENTED** [review]
수고하셨습니다.

**Comment by bluefriday** [comment]
> 수고하셨습니다.

@jmnote 리뷰해주신 내용으로 수정하였습니다.

File: content/ko/docs/topics/kubernetes_distros.md [hunk 9]
```diff
@@ -0,0 +1,97 @@
+---
+title: "쿠버네티스 배포판 가이드"
+description: "특정 쿠버네티스 환경에서 헬름 사용에 대한 정보 정리."
+aliases: ["/docs/kubernetes_distros/"]
```
Comment (bridgetkromhout) [inline content/ko/docs/topics/kubernetes_distros.md:4]: Please remove this line. Aliases to English should not appear in translations, as that will redirect readers from English to a translation.

Comment (bluefriday) [reply content/ko/docs/topics/kubernetes_distros.md:4]: @bridgetkromhout  Yes, I  will remove aliases as well as also in future translations.

**Comment by bluefriday** [comment]
@bridgetkromhout 
Thanks for your review :D
I edited as your comment.

File: content/ko/docs/topics/kubernetes_distros.md [hunk 10]
```diff
@@ -0,0 +1,98 @@
...
+## EKS
+
+헬름은 Amazon Elastic 쿠버네티스 서비스 (Amazon EKS)에서 
+동작한다:[Amazon EKS 와 
+함께 헬름 사용](https://docs.aws.amazon.com/eks/latest/userguide/helm.html).
+
+## GKE
+
+구글의 GKE 호스팅 쿠버네티스 플랫폼은 헬름과 함께 동작하는 것으로 
+알려져 있으며, 추가 구성이 필요하지 않다.
```
[the remark this answers is not in the document]
Comment (bridgetkromhout) [reply content/ko/docs/topics/kubernetes_distros.md]: I have a question - what does "powered by Helm" imply here? The original translation (which you changed here) sounded like it stated Helm can be used with GKE, which is true. We do not want to accidentally imply that Helm "powers" or is the fundamental driving component for GKE itself (which as far as I know is definitely not true).

**Comment by bridgetkromhout** [comment]
LGTM if someone can check on this: https://github.com/helm/helm-www/pull/804#discussion_r488080014 - @bluefriday?

File: content/ko/docs/topics/kubernetes_distros.md (hunk 10 above)
[the remark this answers is not in the document]
Comment (jmnote) [reply content/ko/docs/topics/kubernetes_distros.md]: It doesn't mean 'powered by'. I don't know how to explain to you the subtle differences. It's just more natural in Korean.

**Review by bridgetkromhout — APPROVED** [review]
Thank you!

**Pull request merged**